### PR TITLE
Add title column to configure-sharepoint-content-source.md

### PR DESCRIPTION
**Pull request by dariomws**
Closes https://github.com/MicrosoftDocs/Viva/issues/563

### CODE REVIEW

File: Viva/learning/configure-sharepoint-content-source.md [hunk 1]
```diff
@@ -187,13 +187,20 @@ To add a description column to the document library page, follow these steps:
 3. Name the column ContentDescription.
 4. Add custom descriptions for each item. If no description is supplied, Viva Learning will provide a default message that highlights the content as being from your own SharePoint library.
 
+Next, add a title column.
+
+1. Follow the initial steps to create a column.
+2. Choose **Multiple lines of text**.
+3. Name the column ContentTitle.
+4. Add custom title for each item. If no title is supplied, Viva Learning will pick the file name as the title.
```
Comment (JohanFreelancer9) [inline Viva/learning/configure-sharepoint-content-source.md:195]:
```suggestion
4. Add a custom title for each item. If no title is supplied, Viva Learning will pick the file name as the title.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by scanum** [comment]
@dariomws

**Comment by getalex** [comment]
Title column already present.

**Pull request closed without merging**